README.md

```diff
@@ -256,6 +258,51 @@ Personal API keys only need read permission to Query. When creating a personal A
 
 Once these are set (and you've restarted the app), you should see events flowing into the PostHog dashboard.
 
+### Connecting with MITxOnline
+
+Set up the [mitxonline](https://github.com/mitodl/mitxonline) as indicated there. _Note: A working OpenEdx installation is not necessary for integration with Learn._
+
+Both MITxOnline and MIT Learn repos have keycloak and apisix containers included in their stacks. In connecting the two projects, we want to use the same Keycloak+Apisix container for both projects. Here we assume that the Learn Keycloak+Apisix will be used.
+
+Then, in Learn, set:
+
+```env
+# MIT Learn, backend.local.env
+MITX_ONLINE_BASE_URL=http://mitxonline.odl.local:8013/
+MITX_ONLINE_COURSES_API_URL=http://host.docker.internal:8013/api/v2/courses/
+MITX_ONLINE_PROGRAMS_API_URL=http://host.docker.internal:8013/api/v2/programs/
+```
+
+```env
+# MIT Learn, frontend.local.env
+## This is the default; it should not be overridden
+NEXT_PUBLIC_MITX_ONLINE_BASE_URL=${MITX_ONLINE_BASE_URL}
+```
+
+and in MITxOnline, set:
+
+> [!TIP]
+> Be sure that COMPOSE_PROFILES is NOT set in your .env file; that way, MITxOnline's keycloak and apisix containers will not be used.
+
+```env
+# MITxOnline, .env
+CSRF_COOKIE_DOMAIN=.odl.local
+CORS_ALLOWED_ORIGINS=http://mitxonline.odl.local:8065, http://learn.odl.local:8062
+CSRF_TRUSTED_ORIGINS=http://mitxonline.odl.local:8065, http://learn.odl.local:8062
+
+# For mitol-django-apigateway
+MITOL_APIGATEWAY_DISABLE_MIDDLEWARE=False
+MITOL_APIGATEWAY_USERINFO_CREATE=True
+MITOL_APIGATEWAY_USERINFO_UPDATE=True
+
+# Keycloak settings, these should match mit-learn
+KEYCLOAK_BASE_URL=http://kc.ol.local:8066
+KEYCLOAK_CLIENT_ID=apisix
+KEYCLOAK_CLIENT_SECRET=HckCZXToXfaetbBx0Fo3xbjnC468oMi4 # pragma: allowlist-secret
+KEYCLOAK_DISCOVERY_URL=http://kc.ol.local:8066/realms/ol-local/.well-known/openid-configuration
+KEYCLOAK_REALM_NAME=ol-local
```

Comment:
I could not log in with these settings.   But the backpopulate command works regardless.